views/personal/address-edit.pug

```diff
@@ -1,54 +0,0 @@
-extends ../base
-
-block variables
-  -var title = __('Change your mailing address')
-  -var aptNumber = hasData(body, 'line2') ? body.line2 : hasData(data, 'personal.address.line2') ? data.personal.address.line2 : ''
-  -var streetAddress = hasData(body, 'line1') ? body.line1 : hasData(data, 'personal.address.line1') ? data.personal.address.line1 : ''
-  -var city = hasData(body, 'city') ? body.city : hasData(data, 'personal.address.city') ? data.personal.address.city : ''
-  -var postalCode = hasData(body, 'postalCode') ? body.postalCode : hasData(data, 'personal.address.postalCode') ? data.personal.address.postalCode : ''
-  -var province = hasData(body, 'province') ? body.province : hasData(data, 'personal.address.province') ? data.personal.address.province : ''
-
-
-block content
-
-  h1 #{title}
-
-  div
-    p #{__('Please enter the correct mailing address information below. If a letter is sent to this address, you must be able to receive it.')}
-
-  div
-    form.cra-form.cra-form--lg(method='post')
-      h2 #{__('Your mailing address')}
```

Comment:
This was my favourite form in the whole app 😿

Reply:
I know, I thought about making the PR description just a long apology to you